.github/workflows/create-release.yml

```diff
@@ -0,0 +1,34 @@
+on:
+  push:
+    tags:
+      - '*-*-*'
+
+name: Create release draft
+
+jobs:
+  build:
+    name: Create release from tag
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v3
+
+      - name: use node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: download dependencies
+        run: npm ci
+
+      - name: create release tarball
+        run: ./.github/create-release.sh
```

Comment:
The path is incorrect here: it should just be `./create-release.sh` as it's in the top level, right?

Reply:
I moved this script to the ./github folder because I think it fits better there. 
However, if that should cause problems or something similar, I can move the script back to the top level.